includes/class-wcmp-settings-callbacks.php

```diff
@@ -51,21 +51,22 @@ public function checkbox( $args ) {
 	 */
 	public function text_input( $args ) {
 		extract( $this->normalize_settings_args( $args ) );
-		// echo '<pre>';var_dump($this->normalize_settings_args( $args ));echo '</pre>';
 		if (empty($type)) {
 			$type = 'text';
 		}
 
 		if ($type == 'number') {
-			$width = ($size * 10) + 25;
+			$width = ($size) + 200;
 			$style = "width: {$width}px";
+			$step = "0.01";
```

Comment:
Maak van "0.01" een cont zodat het duidelijker is, wat het is.

Reply:
Done!